src/widgets/QDisassemblyView.cpp

```diff
@@ -1647,11 +1676,19 @@ int QDisassemblyView::line0() const {
 // Desc:
 //------------------------------------------------------------------------------
 int QDisassemblyView::line1() const {
-	if(line1_ == 0) {
+
+	if(!edb::v1::config().show_jump_arrow) {
+
+		// allocate space for register badge
+		// 5 (maximum register name length i can find off) + overhead
```

Comment:
Which of the registers has 5-char name? `MXCSR`? `YMM12`? I would be surprised if they were able to point anywhere ;)
Only GPRs are used for the badges, see the beginning of `drawRegiserBadges()`.

Reply:
Fixed